crime-evidence/src/test/java/uk/gov/justice/laa/crime/evidence/data/builder/TestModelDataBuilder.java

```diff
@@ -64,6 +72,58 @@ public static ApiCalculateEvidenceFeeRequest getApiCalculateEvidenceFeeInvalidRe
                 .withEmstCode("SELF");
     }
 
+    public static ApiUserSession getUserSession() {
+        return new ApiUserSession()
+                .withUserName(TEST_USER_NAME)
+                .withSessionId(UUID.randomUUID().toString());
+    }
+
+    public static ApiApplicantDetails getApiApplicantDetails(boolean isPartner) {
+        return new ApiApplicantDetails()
+                .withId(isPartner ? APPLICANT_ID : PARTNER_ID)
```

Comment:
I think you have the `APPLICANT_ID` and `PARTNER_ID` values here in the ternary the opposite way around to what they should be (if I'm reading the method correctly and how it's being used on line 99)?

Reply:
Adding another thought, it might make things easier to read and comprehend if the `isPartner` param is removed and another method created specifically for `getApiPartnerDetails`? It adds a little duplication but makes it much easier to understand as opposed to passing a bool.

Reply:
Good suggestion, I've added a second method :)